core/src/contracts/stablex_contract.rs

```diff
@@ -80,6 +80,14 @@ pub struct FilteredOrderPage {
     pub next_page_user_offset: u16,
 }
 
+#[derive(thiserror::Error, Debug)]
+pub enum NoopTransactionError {
+    #[error("no account")]
+    NoAccount,
+    #[error("execution error")]
```

Comment:
nit: Include the underlying error in the error message
```suggestion
    #[error("execution error: {0}")]
```